box/src/Box.styles.ts

```diff
@@ -1,9 +1,12 @@
-.wrapper {
+export const wrapper = /*css*/ `
```

Comment:
I didn't see this yesterday
why?
you didn't add a Shadow DOM to this component, did you? so this is making things worse from the styles encapsulation perspective